application/src/main/java/org/opentripplanner/apis/transmodel/TransmodelGraphQLSchema.java

```diff
@@ -1273,7 +1275,7 @@ private GraphQLSchema create() {
             GraphQLArgument
               .newArgument()
               .name("activeDates")
-              .description("Set of ids of active dates to fetch serviceJourneys for.")
+              .description("Set of service dates to fetch serviceJourneys for.")
```

Comment:
Here we use Transmodel/NeTEx terminology:

```suggestion
              .description("Set of operating-days to fetch service-journeys for.")
```

https://enturas.atlassian.net/wiki/spaces/PUBLIC/pages/728727624#framework-OperatingDay

Reply:
Does this mean that operating days are Transmodel terminology, while service dates are OTP internal terminology? If so, I'd like to clean up some more usages in my earlier filter code.

Reply:
Discussed offline.